scripts/one_time_configure_cluster.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Did you already run this script on the related VM?

Reply:
This script is related to configuring cluster with permissions that is required for developer to access registry and exposing the registry route.